LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2019 FIRST
+Copyright (c) 2009-2020 FIRST and other WPILib contributors
```

Comment:
Do you want to remove the year here too?

Reply:
I'm okay keeping it here; it's only one file and helps document the overall development timeframe.